mne/io/edf/tests/test_edf.py

```diff
@@ -124,6 +125,12 @@ def test_edf_data():
         read_raw_edf(broken_fname, exclude=raw.ch_names[:132], preload=True)
 
 
+@pytest.mark.filterwarnings('ignore::RuntimeWarning')
```

Comment:
Do not filter out the warning but rather check that the proper warning is raised using `pytest.warns` and `match`.

```py
    with pytest.warns(RuntimeWarning, match='usefull message documents what are we capturing'):
        call
```